### PR TITLE
Edx 46 code syntax colors

### PR DESCRIPTION
**Pull request by MymmiJ**
> _NOTE TO REVIEWERS_ - please **do not review PRs in the `DRAFT` state**, as the PR may change substantially before it is ready to review. Thanks.

## Description

Updating the `<Code>` element and top code selection nav to match Ably design guidelines 

* [Jira ticket](https://ably.atlassian.net/browse/EDX-46).

## Review

For code reviewers:

- We've replaced the relevant styled components with Tailwind stylings, and extracted classes with multiple functionalities to /styles.css folders
- We've ticketed a refactor of the <HorizontalMenu> part of the codebase, as it is confusing in its current form: https://ably.atlassian.net/browse/EDX-90

For design reviewers:
- https://ably-docs-2-edx-46-code-aug4cp.herokuapp.com/documentation/realtime/channels (for example, please feel free to check other pages)
- Before:
Screenshot 2022-04-08 at 09 34 06
- After: 
Screenshot 2022-04-08 at 09 51 25
- What do you think of the little language label at the top right? It's in the current docs site design, but it requires editors to add an extra newline to make the code look nice and looks awkward when there's no language select menu:
Screenshot 2022-04-08 at 09 36 24 -- either removing the label or improving its appearance will take the same amount of work.
- What do you think of the text when no language is available? Does it take up too much room design-wise? - 
Screenshot 2022-04-08 at 09 52 22

### CODE REVIEW

File: src/components/Menu/TopCodeMenu.js [hunk 1]
```diff
@@ -26,7 +26,7 @@ const TopCodeMenu = ({ languages }) => {
               <LanguageButton language={language} />
             </MenuItem>
           ))}
-        </HorizontalCodeMenu>
+        </TopHorizontalMenu>
       ) : null}
```
Comment (DuncanEdwards) [inline src/components/Menu/TopCodeMenu.js:30]: I know it is not your code, but `<></>` is better than `null` and will type better when we move to TS

File: src/components/Menu/TopCodeMenu.js [hunk 2]
```diff
@@ -12,9 +12,9 @@ const TopCodeMenu = ({ languages }) => {
   const showCodeMenu = languages && languages.length > 1;
   const showDefaultLink = pageLanguage !== DEFAULT_LANGUAGE;
   return (
-    <div className="fixed pt-64 right-0 z-10 w-full items-end">
+    <div className="fixed right-0 z-10 mt-64 w-full items-end">
       {showCodeMenu ? (
-        <HorizontalCodeMenu>
+        <TopHorizontalMenu>
```
Comment (DuncanEdwards) [inline src/components/Menu/TopCodeMenu.js:17]: I know it is not your code, but `<></>` is better than `null` and will type better when we move to TS

**Review by DuncanEdwards — CHANGES_REQUESTED** [review]

File: src/components/Menu/HorizontalMenu.js [hunk 3]
```diff
@@ -1,18 +1,19 @@
 import React from 'react';
 import PropTypes from 'prop-types';
-import { HorizontalCodeMenu } from '.';
+import { HorizontalMenu as HM } from '.';
```
[the remark this answers is not in the document]
Comment (DuncanEdwards) [reply src/components/Menu/HorizontalMenu.js]: Wow, outdated already. I'll mark this as resolved.

File: src/components/Menu/TopCodeMenu.js (hunk 2 above)
Comment (MymmiJ) [reply src/components/Menu/TopCodeMenu.js:17]: Nice! I've never thought of that.

**Review by DuncanEdwards — APPROVED** [review]

**Comment by dpiatek** [comment]
@MymmiJ this looks like a lot more than just the code syntax component :)

**Pull request merged**